qsh/__init__.py

```diff
@@ -326,10 +319,9 @@ def read_ord_log_data(self):
         availability_mask = self.read_byte()
         actions_mask      = self.read_uint16()
 
-        is_add  = True if available(actions_mask, OrdLogEntry.ActionFlag.ADD) else False
-        is_fill = True if available(actions_mask, OrdLogEntry.ActionFlag.FILL) else False
-        is_buy  = True if available(actions_mask, OrdLogEntry.ActionFlag.BUY) else False
-        is_sell = True if available(actions_mask, OrdLogEntry.ActionFlag.SELL) else False
+        is_add  = available(actions_mask, OrdLogEntry.ActionFlag.ADD)
+        is_buy  = available(actions_mask, OrdLogEntry.ActionFlag.BUY)
+        is_sell = available(actions_mask, OrdLogEntry.ActionFlag.SELL)
```

Comment:
The `available` function returns `bool`, so if-checks are unnecessary.
The only concern here is _readability_. I think better readability may be achieved by renaming the function to `is_available`. This general naming convention will be used to show that the function returns a `bool`